### ISSUE TITLE
a minor question about the input dimensions in LA module

### ISSUE THREAD

**Issue by Azai-yx** [HannesStark/protein-localization#1]
Hello HannesStark, 
LightAttention is a really well-designed module for solving signal-peptides classitfition task.
Recently, inspired by your design, I am test it on predicting targeting peptide target-efficiency task. However, the input of LA module is 
self.feature_convolution = nn.Conv1d(embeddings_dim, embeddings_dim, kernel_size, stride=1,
                                             padding=kernel_size // 2)
where the input channel of conv1d is the same dimension as the  embeddings_dim. I have read through your paper but not found a clearly  explaination on this. I guess you may use masked dimensions or something else to make sure the input length is consistent. But here is the problem, my own dataset contains fixed-size peptides, which is 12 amino acids long. Shall I make it masked as u did or just define the first embeddings_dim which is conv1d(input_channel) as 12?

And I would greatly appreciate it if you could offer any insights or suggestions on my prediction tasks which turned classification task(the location prediction which this work focused on) into regression tasks( where my dataset is a bunch of scalar value).

thanks!
